cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBridgeClusterTest.java

```diff
@@ -155,6 +161,9 @@ public void testDefaultValues() {
         assertThat(kbc.readinessProbeOptions.getTimeoutSeconds(), is(5));
         assertThat(kbc.livenessProbeOptions.getInitialDelaySeconds(), is(15));
         assertThat(kbc.livenessProbeOptions.getTimeoutSeconds(), is(5));
+        assertThat(kbc.getKafkaBridgeConsumer().isEnabled(), is(true));
+        assertThat(kbc.getKafkaBridgeConsumer().getTimeoutSeconds(), is(-1L));
```

Comment:
You should really test the environment variables having the right values here. Not the internal fields I think.

Reply:
`testDefaultValues()` only tests default values for the rest of the props, so wouldn't it be wrong to start testing env values here?

I can definitely add the test of the envs in other suitable tests in the class @scholzj :)

Reply:
* I guess this now throws an NPE as the default value it would return is null?
* But it is also not needed. As you pointed out, you test the environment variables. so this test does not add anything because if the environment variables are set correctly, then:
    * We do not really care about these objects that much
    * We know from the environment variables that these values are there and do not need to test them.


It is not like this check would be somehow harmful, but removing the unnecessary getters from the production code would make it a bit leaner and smaller.

Reply:
Will remove this test and getters